cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -21,7 +21,14 @@
     {%- endfor %}
 
     {% for block in page.content -%}
-        {{ render_block.render(block, loop.index) }}
+        {% if block.block_type == 'feedback' %}
```

Comment:
This block of code is repeated in every page template (and similarly in #2415 [here](https://github.com/cfpb/cfgov-refresh/pull/2415/files#diff-45f9e68fb5117558b28c78af907fac50R24)). Can you pull this out somehow to reduce this code duplication, either here or in that other PR?